e2e/cypress/integration/enterprise/cloud/billing/subscriptions_free_trial_spec.js

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+describe('System Console - Subscriptions section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+
+        gotoSubscriptionScreen();
+    });
+
+    beforeEach(() => {
+        // # Click on close button of subscribe window if exist
+        cy.get('body').then(($body) => {
+            if ($body.find('#closeIcon').length > 0) {
+                cy.get('#closeIcon').parent().click();
+            }
+        });
+    });
+
+    it('MM-T4118 - Visibility of Trail tag on Subscription screen', () => {
+        // * Check for visibility of Trail tag
+        cy.contains('span', 'Trial', {timeout: 10000}).should('be.visible');
+    });
```

Comment:
Feels like unit test is more appropriate with this kind of verification. I don't think it's worthy of having its own Zephyr test case (ID) honestly. E2E is expensive (time, cost, maintenance, etc) and we'd like to make sure that we're adding test criteria according to our acceptance/business requirements.

Maybe add more content check on features included? And even combine MM-T4121 and MM-T4121 as part of content verification?

@stevemudie Could you please check each `it` block once again just to make sure that each has Zephyr test case (ID) and are valid test criteria? Thank you!